packages/client3/pages/pools/[address]/index.page.tsx

```diff
@@ -293,11 +292,12 @@ export default function PoolPage() {
           </Heading>
 
           {error ? (
-            <HelperText isError>
-              There was a problem fetching data on this pool. Shown data may be
-              outdated.
-            </HelperText>
-          ) : null}
+            <></>
+          ) : // <HelperText isError>
+          //   There was a problem fetching data on this pool. Shown data may be
+          //   outdated.
+          // </HelperText>
+          null}
```

Comment:
Old component used by goldfinch. I commented instead of deleting it for future reference.